basyx/aas/adapter/aasx.py

```diff
@@ -839,6 +851,14 @@ def get_sha256(self, name: str) -> bytes:
     def write_file(self, name: str, file: IO[bytes]) -> None:
         file.write(self._store[self._name_map[name][0]])
 
+    def delete_file(self, name: str) -> None:
+        hash: bytes = self._name_map[name][0]
```

Comment:
Could you add the motivation you gave in the PR here as docstring in order to make it more easily understandable?